services/wallet/downloader.go

```diff
@@ -234,6 +240,26 @@ func (d *ERC20TransfersDownloader) transfersFromLogs(parent context.Context, log
 	return concurrent.Get(), concurrent.Error()
 }
 
+func (d *ERC20TransfersDownloader) blocksFromLogs(parent context.Context, logs []types.Log, address common.Address) ([]*big.Int, error) {
+	concurrent := NewConcurrentDownloader(parent)
+	for i := range logs {
```

Comment:
```go
for _, log := range logs {

}
```

Reply:
this is actually copied code, I will change it but I kind of can't take credit for it https://github.com/status-im/status-go/blob/0b403ce25bf25be672f9b39ba97384cb2b72b442/services/wallet/downloader.go#L215 :)